### PR TITLE
Adding windows artifact urls

### PR DESCRIPTION
**Pull request by DominikDary**
## High-level description

Provide windows artifact download locations.

### CODE REVIEW

**Comment by DominikDary** [comment]
@mesosphere-mergebot changelog-not-required readme update

**Comment by DominikDary** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by DominikDary** [comment]
@mesosphere-mergebot request-review @jongiddy

File: README.md [hunk 1]
```diff
@@ -35,8 +35,11 @@ To find the git SHA of any given release, check the latest commit in the version
 | Release Type | URL Pattern |
 |--------------|--------------------|
 | Latest Stable| `https://downloads.dcos.io/dcos/stable/dcos_generate_config.sh` |
-| Latest Master | `https://downloads.dcos.io/dcos/testing/master/dcos_generate_config.sh` |
-| Specific PR, Latest Build	| `https://downloads.dcos.io/dcos/testing/pull/<github-pr-number>/dcos_generate_config.sh` |
+| Latest Master Linux| `https://downloads.dcos.io/dcos/testing/master/dcos_generate_config.sh` |
+| Latest Master Windows| `https://downloads.dcos.io/dcos/testing/master/windows/dcos_generate_config_win.sh` |
+| Specific PR, Latest Build	Linux| `https://downloads.dcos.io/dcos/testing/pull/<github-pr-number>/dcos_generate_config.sh` |
+| Specific PR, Latest Build	Windows| `https://downloads.dcos.io/dcos/testing/pull/<github-pr-number>/windows/dcos_generate_config_win.sh` |
```
Comment (jongiddy) [inline README.md]: With the addition of the OS, these last two sound wrong. They should be:
```
Latest Build of Specific PR Linux
```
or
```
Specific PR Linux (Latest Build)
```

Comment (DominikDary) [reply README.md]: updated it, thanks for the feedback.

**Comment by DominikDary** [comment]
@mesosphere-mergebot request-review  @jongiddy

**Pull request merged**